src/client/Microsoft.Identity.Client/AppConfig/UserAssignedIdType.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.AppConfig
+{
+    /// <summary>
+    /// Enum to define types of user assigned identities supported by MSAL.
+    /// </summary>
+    public enum UserAssignedIdType
```

Comment:
See my comment about using objects. I don't think this is needed, or at least not publicly.

Reply:
Updated scope to internal